client/src/App.js

```diff
@@ -9,22 +9,34 @@ import Notes from "./components/Notes/Notes";
 import CourseUpload from "./components/Admin/CourseUpload/CourseUpload";
 import NotesIcon from "./components/NotesIcon/NotesIcon";
 import CourseUpdate from "./components/Admin/CourseUpload/CourseUpdate";
+import { Auth0Provider } from "@auth0/auth0-react";
+import Protected from "./Protected";
 
 function App() {
   return (
-    <>
+    <Auth0Provider
+      domain="dev-ynlvngvzhos0c4m0.us.auth0.com"
```

Comment:
These values can be added in the .env files.

Reply:
done!